tests/autodiff/custom-intrinsic-1.slang

```diff
@@ -1,7 +1,6 @@
 //TEST(compute, vulkan):COMPARE_COMPUTE_EX:-vk -compute -shaderobj -output-using-type
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -shaderobj -output-using-type
-// slang-test/WGPU: IR opcode during code emit #5263
-//DISABLE_TEST(compute):COMPARE_COMPUTE_EX:-wgpu
+//TEST(compute):COMPARE_COMPUTE_EX:-wgpu
```

Comment:
I think this line should be removed.
The line#2 will synthesize one for wgpu.
If we want to explicitly have one for wgpu, it will have to be more like
```
//TEST(compute):COMPARE_COMPUTE_EX:-wgpu -compute -shaderobj -output-using-type
```

Reply:
BTW, CI is failing for this reason.

Reply:
thanks, fixed